cns/restserver/ipam.go

```diff
@@ -20,6 +20,7 @@ import (
 var (
 	ErrStoreEmpty       = errors.New("empty endpoint state store")
 	ErrParsePodIPFailed = errors.New("failed to parse pod's ip")
+	ErrNoPoolIPs        = errors.New("no NCs found on the NNC so no IPs are in the pool")
```

Comment:
This error is returned when we have no NCs, and that's _all_ that it can definitively say. 
Speculating about the state of anything not immediately related to the cause of the error is bad - it's actively misleading to the DRI reading logs and trying to root cause an issue.

Reply:
I can change it so that it only talks about the NC count but separate question: 

is it possible for there to be IPs in the pool without an NC?

Reply:
No, that shouldn't be possible without a lot of other things going wrong

Reply:
@rjdenney Good question to have though, since questions like that are usually what help hammer out good error messages :)